### DISCUSSION TITLE
Is there a way to bypass Max b-frame limit (5) set by necvncc encoder (i assume its by necvncc)

### DISCUSSION THREAD

**Discussion by hmm-mm** [staxrip/staxrip#1422] [Q&A]
So i am encoding a h265 hdr 4k video using necvncc on RTX3060, and i set the bframes to 8 (i dont mind the extra encoding time)
but during the encoding the console replied "Max B frames are 5 frames." and continued encoding with 5 bframes. is there a way to bypass this limit with any tweaking in registry or changing any system files or faking my card to appear newer gen (ik there can be issues and instability with fake card names)?**Also if i messed some settings up, which is in turn limiting my max B frames that can be set, can anyone please point it out**

i am not asking for enabling a feature that is unsupported by my card like for eg.'weighted prediction with B frames', i just wanna increase the number of b-frames from 5. below is the log of my encoding settings. 

--------------------------- Video encoding ---------------------------

NVEncC 7.57

C:\Users\%User%\Downloads\StaxRip-v2.41.6-x64\Apps\Encoders\NVEncC\NVEncC64.exe --avhw --cqp 18:22:26 --codec h265 --preset P7 --output-depth 10 --profile main10 --tier high --level 5.1 --qp-init 1 --aq --aq-temporal --bref-mode each --bframes 8 --ref 5 --lookahead 20 --lookahead-level 3 --slices 4 --tf-level 4 --weightp --colormatrix bt709 --colorprim bt709 --transfer bt709 -i "C:\Users\%%\video.mkv" -o "C:\Users\%%\video_new_out.h265"

-------------------------------------------------------------------------------------------------------------------------------------------------------

**Comment by Dendraspis** [comment]
StaxRip is just the GUI for encoders like `NVEncC`.

I suggest heading over to  https://github.com/rigaya/NVEnc  and ask there if something like that is possible, but I guess this limits are hardware related.
